core/src/main/java/org/elasticsearch/search/aggregations/BaseAggregationBuilder.java

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
Hmmm. I didn't look at that. I'll have a look! I'd love to not need the interface.